.github/workflows/docs.yml

```diff
@@ -35,19 +35,42 @@ jobs:
       - uses: dtolnay/rust-toolchain@7b1c307e0dcbda6122208f10795a713336a9b35a
         with:
           toolchain: stable
+      - uses: Swatinem/rust-cache@23bce251a8cd2ffc3c1075eaa2367cf899916d84
```

Comment:
Although this is unrelated, I'm just not sure why this particular workflow doesn't have cache 🤔

Reply:
I think we might have just forgotten to add one